src/Pikachu.java

```diff
@@ -0,0 +1,7 @@
+public class Pikachu extends Pokemon {
```

Comment:
Dit is niet perse fout, maar je wilt lievers iets generiekere programmeren. Dus hier een Electric pokemon van maken. En dan een 'private String name' property mee geven. Nu kan deze classe enkel Pikachu aanmaken. Tenzij dat is wat je wilt in je applicatie, maar pokemon heeft niet voor al haar pokemon een aparte class aangemakt in hun code.